test/format/backup.c

```diff
@@ -41,10 +41,34 @@ check_copy(void)
     size_t len;
     char *path;
 
-    len = strlen(g.home) + strlen("BACKUP") + 2;
+    len = WT_MAX(strlen(BACKUP_INFO_FILE), strlen(BACKUP_INFO_FILE_TMP));
+    len = WT_MAX(len, strlen("BACKUP"));
+    len += strlen(g.home) + 2;
     path = dmalloc(len);
+    /*
+     * Remove any backup info files that exist. We're about the run recovery in the backup directory
+     * so we cannot use the backup directory after that restart. We must remove the files before the
+     * restart to avoid a window where they could exist and the backup directory has had recovery
+     * run.
+     */
+    testutil_check(__wt_snprintf(path, len, "%s/%s", g.home, BACKUP_INFO_FILE_TMP));
+    ret = unlink(path);
+    /* It is fine if the file does not exist. */
+    if (ret == 0 || errno == ENOENT)
```

Comment:
Hmm, I have to take into account the successful case, or ret == -1 and errno == ENOENT. If I just remove the `ret == 0` I have to add some other conditional to avoid the `testutil_die()` from another error.

Reply:
You're right!  never mind my comment.